sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -907,6 +907,12 @@ object SQLConf {
     .stringConf
     .createOptional
 
+  val FORCE_DELETE_TEMP_CHECKPOINT_LOCATION =
+    buildConf("spark.sql.streaming.forceDeleteTempCheckpointLocation")
```

Comment:
Let us follow the other boolean conf naming convention? 

`spark.sql.streaming.forceDeleteTempCheckpointLocation.enabled`?

Reply:
cc @Ngone51 could you submit a PR to make a change?

Reply:
Submitted #26981